hypervisor/arch/x86/guest/vioapic.c

```diff
@@ -393,14 +393,10 @@ vioapic_write(struct vioapic *vioapic, uint32_t addr, uint32_t data)
 			vioapic_send_intr(vioapic, pin);
 		}
 
-		/* remap for active: interrupt mask -> unmask
-		 * remap for deactive: interrupt mask & vector set to 0
+		/*
+		 * remap for any update to RTL
```

Comment:
Why it is called RTL ?

Reply:
It's a mistake it should be RTE (redir table entry)